Connectors/Rhino/Speckle.Connectors.GrasshopperShared/Parameters/SpeckleBlockInstanceWrapper.cs

```diff
@@ -158,31 +141,34 @@ public void Bake(RhinoDoc doc, List<Guid> objIds, int bakeLayerIndex = -1)
     }
   }
 
-  public SpeckleBlockInstanceWrapper DeepCopy() =>
-    new()
+  public override SpeckleObjectWrapper DeepCopy() =>
+    new SpeckleBlockInstanceWrapper // covariant return types to satisfy inheritance chain
     {
       Base = InstanceProxy.ShallowCopy(),
-      Definition = Definition?.DeepCopy(),
-      Transform = _transform,
-      Properties = Properties,
+      GeometryBase = null, // safe default. GeometryBase is null for instances?
```

Comment:
We need this GeometryBase property on instances. It corresponds to an InstanceReferenceGeometry object that inherits from GeometryBase in rhinocommon

Reply:
Updated comment, thanks. [docs: GeometryBasenote on SpeckleBlockInstanceWrapper](https://github.com/specklesystems/speckle-sharp-connectors/pull/917/commits/84c45d97f6a3909ddb8153c20964741e0ebd9599)